common/ethnicities/wc_ethnicities_vrykul.txt

```diff
@@ -0,0 +1,452 @@
+	}
+	beards = {
+		1 = { name = vrykul_beards 		range = { 0.0 1.0 } }
+	}
+	special_eyebrows = {
+        1 = {  name = no_eyebrows             range = { 0 1 }     }
+	}
+	face_detail_temple_def = {
+        1 = {  name = temple_def             range = { 0.7 0.9 }     }
+	}
+	expression_brow_wrinkles = {
+        1 = {  name = brow_wrinkles_01             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_02             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_03             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_04            range = { 0.5 1 }     }
+	}
+	expression_eye_wrinkles = {
+        1 = {  name = eye_wrinkles_01             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_02             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_03             range = { 0 1 }     }
+	}
+	expression_forehead_wrinkles = {
+        1 = {  name = forehead_wrinkles_01             range = { 0.1 1 }     }
+		1 = {  name = forehead_wrinkles_02             range = { 0.1 1 }     }
+		1 = {  name = forehead_wrinkles_03             range = { 0.1 1 }     }
+	}
+	complexion = {
+        1 = {  name = complexion_1             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_3             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_4             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_5             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_6             range = { 0.43 0.43 }     }
+	}
+}
+
+frost_vrykul_ethnicity = {
+	template = "vrykul_ethnicity"
+	
+	skin_color = {
+		1 = { 0.700 0.349 0.722 0.467 } # Blue
+	}
+	eye_color = {
+		1 = { 0.639 0.447 0.700 0.655 } # Blue
+	}
+	hair_color = {
+		1 = { 0.675 0.102 0.675 0.239 } # White/Light Blue
+		1 = { 0.686 0.443 0.722 0.675 } # Blue
+	}
+	
+	###Glowy Eyes
+	special_eyes = {
+		1 = { name = magic_eyes      range = { 0 1 } }
+	}
+}
+
+kvaldir_ethnicity = {
+	template = "vrykul_ethnicity"
+	
+	skin_color = {
+		1 = { 0.349 0.333 0.522 0.431 } # Yellow-Green to Green
+	}
+	eye_color = {
+		1 = { 0.557 0.176 0.557 0.424 } # White to Blue
+	}
+	hair_color = {
+		1 = { 0.314 0.820 0.365 0.949 } # Dark Green
+		1 = { 0.004 0.910 0.231 0.937 } # Red & Brown
+	}
+	
+	###Glowy Eyes
+	special_eyes = {
+		1 = { name = magic_eyes      range = { 0 1 } }
+	}
+}
+
+drust_ethnicity = {
+	template = "vrykul_ethnicity"
+	
+	skin_color = {
+		2 = { 0.737 0.294 0.737 0.337 } # Gray/Purple
+		1 = { 0.153 0.008 0.153 0.008 } # Dark
+	}
+	eye_color = {
+		1 = { 0.603 0.510 0.647 0.627 } # Cyan/Blue
+	}
+	hair_color = {
+		1 = { 0.812 0.024 0.812 0.055 } # Gray/White
+	}
+	
+	###Glowy Eyes
+	special_eyes = {
+		1 = { name = magic_eyes      range = { 0 1 } }
+	}
+	
+	###Other	
+	hairstyles = {
+		1 = { name = northern_hairstyles 		range = { 0.0 1.0 } }
+		5 = { name = no_hairstyles 		range = { 0.0 1.0 } }
+	}
+	beards = {
+		1 = { name = northern_beards 		range = { 0.0 1.0 } }
+		5 = { name = no_beard 		range = { 0.0 1.0 } }
+	}
```

Comment:
Genetically bald?
If not, consider making them more clean-shaved and bald in the portrait modifiers.

Reply:
> 
> 
> Genetically bald?
> If not, consider making them more clean-shaved and bald in the portrait modifiers.

I don't know why it's showing these for you, I've already changed this part so they use `vrykul_hairstyles` and `vrykul_beards`.

Reply:
> I don't know why it's showing these for you, I've already changed this part so they use `vrykul_hairstyles` and `vrykul_beards`.

No, I just checked it in the file.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108603064-11d43600-73bf-11eb-884a-a62ff830f10b.png)

</details>